tests/ccl_test_core.py

```diff
@@ -183,9 +183,20 @@ def test_cosmology_init():
     
     # Make sure error raised if incorrect type of Parameters object passed
     assert_raises(TypeError, ccl.Cosmology, params=params.parameters)
+    assert_raises(TypeError, ccl.Cosmology, params="xx")
```

Comment:
Just to be consistent with the rest why not use `params='x'`?

Reply:
I'll change this now. Thanks!